vanilla/vanilla
# Fix Keystone/Boilerplate build

Closes https://github.com/vanilla/support/issues/2109

**Steps to test:**
- Make sure you have [Vanilla CLI installed](https://success.vanillaforums.com/kb/articles/241-vanilla-cli-installation)
- Checkout this branch
- cd into `addons/themes/keystone`, run `vanilla build`, make sure it works
- cd into `addons/themes/theme-boilerplate`, run `vanilla build`, make sure it works
- run `yarn build`, make sure it works
